owners/test/index.test.js

```diff
@@ -48,6 +48,7 @@ const pullRequest35 = require('./fixtures/pulls/pull_request.35');
 nock.disableNetConnect();
 jest.setTimeout(30000);
 
+const GITHUB_REPO = 'erwinmombay/github-owners-bot-test-repo';
 const GITHUB_REPO_DIR = '/Users/erwinm/dev/github-owners-bot-test-repo';
```

Comment:
Maybe change this to a generic fake username (as opposed to referencing a real LDAP)?

Reply:
This is what's in use throughout the file (and there's a handful of other real usernames in tests in general). I filed #567 and can address this in a separate PR